requirements.txt

```diff
@@ -0,0 +1 @@
+flask>=0.10.1
```

Comment:
requirements.txt should pin versions. Probably exact versions that the app was tested with. You run a real risk here of suddenly trying to use flask 1.0 or flask 0.11, either of which might change something which breaks the app.